src/pl/senla/hotel/service/ServiceFacilityImpl.java

```diff
@@ -15,20 +15,20 @@
 @AppComponent
 public class ServiceFacilityImpl implements ServiceFacility{
 
-    @GetInstance(beanName = "RepositoryFacilityCollection")
-    private Repository<HotelFacility> repositoryHotelFacility;
-    @GetInstance(beanName = "RepositoryRoomCollection")
-    private Repository<Room> repositoryRoom;
+    @GetInstance(beanName = "DaoFacilityCollection")
+    private GenericDao<HotelFacility> daoHotelFacility;
+    @GetInstance(beanName = "DaoRoomCollection")
+    private GenericDao<Room> daoRoom;
 
     public ServiceFacilityImpl() {}
 
     @Override
     public List<HotelFacility> readAll() {
-        if(repositoryHotelFacility.readAll() == null || repositoryHotelFacility.readAll().isEmpty()){
+        if(daoHotelFacility.readAll() == null || daoHotelFacility.readAll().isEmpty()){
             System.out.println(ERROR_READ_ALL_HOTEL_FACILITY);
```

Comment:
why it's an error? it's one of the possible flows of how your program should work